js/templates/listingCard.html

```diff
@@ -42,7 +42,7 @@
   </div>
   <% if (ob.vendor) { %>
   <a class="userIconWrapper js-userName">
-    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? `  📍  ${ob.vendor.location}` : ''}`;
+    <% const vendorTip = `${ob.vendor.name}${ob.vendor.location ? ` ${ob.vendor.location}` : ''}`;
```

Comment:
> I think we should remove the location from displaying in the tooltip if that is the case.

Unless I'm misinterpreting, @morebrownies said to remove the location entirely if we can't show the emoji.